### PR TITLE
fix work forcing www/no-www with https

### LINKED ISSUE #52: Remove only-http from www-normalizing redirects
Most users probably aren't using both http and https, but if they are it's possibly confusing/misleading why these redirects only apply to http requests.

So, as mentioned in #49 the `RewriteCond %{HTTPS} =on` conditions should be removed before the next major release.

### PR DESCRIPTION
**Pull request by xtratio**
Problem: Options "Suppressing / Forcing the `www.` at the beginning of URLs" only work with http.
Hope this will be helpful.

Fixes #52

### CODE REVIEW

**Comment by lucaspetter** [comment]
Agreed. This should be merged. :+1:

Rewrites that force/suppress the `www` should be independent of rewrites that force/suppress `https`. It should be possible to force/suppress `www` regardless of whether the protocol is `https` or `http`. This commit fixes the current dependence on `http` for forcing/suppressing `www`.

I came here to make the exact same pull request, but am glad to see someone else has already done it.

**Comment by XhmikosR** [comment]
Discussion on this: https://github.com/h5bp/server-configs-apache/issues/52

**Comment by ManasN** [comment]
This should be merged along with a comment, really. I knew they have some intention to put it there but couldn't figure out why, now that #52 makes it clear, it will be helpful if we keep this in the .htaccess.

File: src/rewrites/rewrite_www.conf [hunk 1]
```diff
@@ -22,7 +22,6 @@
 
 <IfModule mod_rewrite.c>
     RewriteEngine On
-    RewriteCond %{HTTPS} !=on
```
Comment (LeoColomb) [inline src/rewrites/rewrite_www.conf]: @xtratio Could you add a comment in the heading part as @ManasN (https://github.com/h5bp/server-configs-apache/pull/84#issuecomment-289069219)?

**Comment by LeoColomb** [comment]
Closed #84 via fc747bbdf0a0c224ec08d8b925f33671e4d5046d.

**Comment by lucaspetter** [comment]
Thank you, @LeoColomb !

**Pull request closed without merging**